### ISSUE TITLE
Could you provide full results for download?

### ISSUE THREAD

**Issue by zhyanlin** [JinLabBioinfo/DeepLoop#4]
Hello,

I saw you provided top300k results (https://hiview.case.edu/ssz20/tmp.HiCorr.ref/DeepLoop_top300K.tar.gz) for downloading and full results for online view (https://hiview.case.edu/public/DeepLoop/). Is there any chance that I can download some of your full results?

**Comment by dylan-plummer** [comment]
Yes, you can find .cool files for each dataset here: [GSE167200](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE167200)

See the `GSE167200_39cool.tar.gz` and `GSE167200_Readme_39cool.txt` files.

If you prefer the same format as the top300k results, you can use the [convert_from_cooler.py](https://github.com/JinLabBioinfo/DeepLoop/blob/97e6276789b6551af4c484fb9a6d529675050c0c/utils/convert_from_cooler.py) script.
